packages/webpack/src/builder.js

```diff
@@ -27,9 +27,11 @@ export class WebpackBundler {
     this.devMiddleware = {}
     this.hotMiddleware = {}
 
-    // Initialize shared FS and Cache
+    // Initialize shared MFS for dev
     if (this.context.options.dev) {
       this.mfs = new MFS()
+      this.mfs.exists = function (...args) { return Promise.resolve(this.existsSync(...args)) }
```

Comment:
It seemed that the rewrited `exists` and  `readFile` are not used in `loadResources`.

BTW: `memory-fs` already provided `exists` with `callback` version, so maybe we just need a pify ?

```suggestion
this.mfs.exists = pify(this.mfs.exists)
```

Reply:
I've tried pify. It is not a standard callback (error first) and even with passing proper options it was still buggy. So i ended up with this solution that was the safest one!